docs/Run-Cluster-Genesis.rst

```diff
@@ -289,6 +362,9 @@ Each file must be formatted according to the following rules:
       Column headers must be delimited by at least two spaces. Single spaces
       will be considered a continuation of a single column header (e.g. "mac
       address" is one column, but "mac address  vlan" would be two).
+    * If a header is provided, it must include a separator row consisting of
```

Comment:
I think this bullet, and the ones before and after describing the header, should be nested under '* MAC addresses and ports are listed in a tabular format.'